doc/neps/nep-0041-improved-dtype-support.rst

```diff
@@ -156,6 +159,13 @@ Further, the public API will be designed in a way that is extensible in the futu
    as much as possible. The public API should be an identical, but limited,
    version of the C-API used for the internal NumPy datatypes.
 
+The datatype system may be targeted to work with NumPy arrays,
+for example by providing strided-loops, but should avoid using the array-object.
```

Comment:
Not sure if this clarification is needed, but...
```suggestion
for example by providing strided-loops, but should avoid direct interactions with the array-object (typically `np.ndarray` instances).
```

Reply:
Can't hurt, thanks!